assignment/assignment-bundles/resources/assignment.properties

```diff
@@ -898,6 +902,7 @@ content_review.pending.info = This attachment has been submitted and is pending
 content_review.error = An unknown error occurred. The originality review for this attachment is not available.
 content_review.error.createAssignment=An error with {0} has occurred while creating this assignment. {1} has saved the assignment in draft mode. Please try posting this assignment again later. {2}
 content_review.note=<div><br /><em>NOTE:</em><ul><li>When submitting attachments, students should only use these file types: {0}.</li> <li>Students should always save files with the appropriate extension.</li> </ul></div>
+content_review.studentNote = Only use file types: Microsoft Word (.doc, .docx), PowerPoint (.ppt, .pptx), PostScript (.ps), PDF (.pdf), HTML (.html), OpenOffice (ODT), Hangul (HWP), rich or plain text (.rtf, .txt).</li><li>Always include file extension.</li><li>Maximum file size is 20 Mb. The document must contain more than 20 words.</li></ol>
```

Comment:
Should paramaterize the file type list passing in AssignmentAction.getContentReviewAcceptedFileTypesMessage()